easybuild/easyconfigs/c/Cbc/Cbc-2.10.12-foss-2024a.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'ConfigureMake'
+
+name = 'Cbc'
+version = '2.10.12'
+
+homepage = 'https://github.com/coin-or/Cbc'
+description = """Cbc (Coin-or branch and cut) is an open-source mixed integer linear programming
+solver written in C++. It can be used as a callable library or using a
+stand-alone executable."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/coin-or/%(name)s/archive/refs/tags/releases']
+sources = ['%(version)s.tar.gz']
+checksums = ['9ed71e4b61668462fc3794c102e26b4bb01a047efbbbcbd69ae7bde1f04f46a8']
+
+builddependencies = [
+    ('Autotools', '20231222'),
+    ('Doxygen', '1.11.0'),
+    ('pkgconf', '2.2.0'),
+]
+dependencies = [
+    ('METIS', '5.1.0'),
+    ('MUMPS', '5.7.2', '-metis'),
+    ('CoinUtils', '2.11.10'),
```

Comment:
This (and the others) need to be updated to reflect your version bumos

Reply:
Yeah, thanks. I must have been doing just that when you were writing you comment :-) It should nave have been commit.